### DISCUSSION TITLE
Where can I find loco's websocket guide

### DISCUSSION THREAD

**Discussion by RustLoverTheCoder** [loco-rs/loco#323] [Q&A] [answered]
Where can I find loco's websocket guide

**Comment by kaplanelad** [comment, marked as answer]
@RustLoverTheCoder, at the moment, we don't have a specific example illustrating the implementation of WebSocket. We have an  issue #208, that  tagged for implementation in the upcoming [0.2.0](https://github.com/loco-rs/loco/milestone/6) milestone (next release)

If you have any insights or suggestions to share before we commence development, your input would be greatly appreciated in the ticket

**Comment by Daniel-Xu** [comment]
I hope this project will be the Phoenix for Elixir
